kitsune/products/views.py

```diff
@@ -18,12 +19,20 @@ def product_list(request):
     return render(request, template, {"products": products})
 
 
+def _get_aaq_product_key(products, slug):
```

Comment:
nit: I'd just take a single slug argument here, and directly reference `aaq_config.products` in the function